manager/report_handler.py

```diff
@@ -254,7 +251,7 @@ def _get_top_cves_by_cvss(cvss_from, cvss_to, count_query, limit=3):
         # pylint: disable=singleton-comparison
         rules = (CveRuleMapping.select(CveRuleMapping.cve_id, CveRuleMapping.rule_id)
                  .join(InsightsRule, on=((CveRuleMapping.rule_id == InsightsRule.id)
-                                         & (InsightsRule.active == True) & (~InsightsRule.rule_only))))
+                                         & (InsightsRule.active == True) & (InsightsRule.rule_only == False))))
```

Comment:
You should remove the `& (InsightsRule.rule_only == False))`. Because the `rule_only` is not used completely in the `count_query` on line 96.